data/json/requirements/materials.json

```diff
@@ -559,6 +559,57 @@
     "//": "Both wax types, for when the recipe doesn't really care.",
     "components": [ [ [ "wax", 1 ], [ "wax_paraffin", 1 ] ] ]
   },
+  {
+    "id": "plastic_bottles_any",
+    "type": "requirement",
+    "//": "Any plastic bottle, no matter of it's size.",
+    "components": [
+      [
+        [ "jug_plastic", 1 ],
+        [ "bottle_twoliter", 1 ],
+        [ "bottle_plastic", 1 ],
+        [ "bottle_folding", 1 ],
+        [ "bottle_bathroom", 1 ],
+        [ "bottle_plastic_seasoning", 1 ],
+        [ "bottle_plastic_small", 1 ],
+        [ "bottle_plastic_seasoning_small", 1 ]
```

Comment:
Another suggestion for "any plastic bottles" of the top of my head:
* `bottle_plastic_pill_prescription`

Reply:
i didn't add them, because their 250 ml size is not because they should be 250 ml, but because they try to be able to fit at least 1 charge of water, and this causes all sorts of weirdness with watertight containers being at least 250 ml